keepercommander/importer/lastpass/fetcher.py

```diff
@@ -40,15 +42,31 @@ def logout(session, web_client=http):
 
 
 def fetch(session, web_client=http):
-    response = web_client.get('https://lastpass.com/getaccts.php?mobile=1&b64=1&hash=0.0&hasplugin=3.0.23&requestsrc=android',
-                              cookies={'PHPSESSID': session.id})
+    url = 'https://lastpass.com/getaccts.php' + query_string
+    response = web_client.get(url, cookies={'PHPSESSID': session.id})
 
     if response.status_code != requests.codes.ok:
         raise NetworkError()
 
     return blob.Blob(decode_blob(response.content), session.key_iteration_count)
 
 
+def fetch_shared_folder_members(session, shareid, web_client=http):
+    url = 'https://lastpass.com/getSharedFolderMembers.php' + query_string + f'&shareid={shareid}'
+    response = web_client.get(url, cookies={'PHPSESSID': session.id})
+
+    if response.status_code != requests.codes.ok:
```

Comment:
I think we need to be sure that this code does not throw any error.
It's better this method does nothing that cancels the whole import